src/js/actions/shapes.js

```diff
@@ -42,17 +42,18 @@ define(function (require, exports) {
         strings = require("i18n!nls/strings");
 
     /**
+     * Merges shape specific options into given options
      * play/batchPlay options that allow the canvas to be continually updated, 
      * and history state to be consolidated 
      *
      * @private
+     * @param {object} options
      * @param {object} documentRef  a reference to the document 
      * @param {string} name localized name to put into the history state
-     * @param {boolean=} coalesce Whether to coalesce this operations history state
      * @return {object} options
      */
-    var _options = function (documentRef, name, coalesce) {
-        return {
+    var _mergeOptions = function (options, documentRef, name) {
```

Comment:
This function weird me out a little.  I feel like this is the translation between the "options" being passed around by our methods, and the "options" sent to photoshop.  If that's the case, then maybe it should really "merge" at all.